cmd/psqldef/tests.yml

```diff
@@ -479,3 +479,37 @@ AddArrayColumn:
     );
   output: |
     ALTER TABLE "public"."users" ADD COLUMN "name" integer[];
+AddForeignKeyWithAlter:
+  current: |
+    CREATE TABLE users (id BIGINT PRIMARY KEY);
+    CREATE TABLE posts (
+      content text,
+      user_id bigint
+    );
+  desired: |
+    CREATE TABLE users (id BIGINT PRIMARY KEY);
+    CREATE TABLE posts (
+      content text,
+      user_id bigint
+    );
+    ALTER TABLE ONLY public.posts ADD CONSTRAINT posts_ibfk_1 FOREIGN KEY (user_id) REFERENCES users (id) ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
Is it intentional that the parser accepts `ALTER TABLE ONLY` but not `ALTER TABLE`? I expected `mysqldef` to accept the following schema:

```
CREATE TABLE `error_logs`(
  `id` BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  `credential_id` BIGINT NOT NULL,
  `message` TEXT CHARACTER SET utf8mb4 NOT NULL,
  `created_at` DATETIME NOT NULL,
  `resolved` TINYINT(1) NOT NULL);
ALTER TABLE `error_logs`
  ADD CONSTRAINT `fkey`
  FOREIGN KEY(`credential_id`)
  REFERENCES `credentials`(`id`)
  ON DELETE CASCADE
  ON UPDATE RESTRICT;
```

However it produced the following error.

```
found syntax error when parsing DDL "ALTER TABLE `error_logs`
  ADD CONSTRAINT `fkey`
  FOREIGN KEY(`credential_id`)
  REFERENCES `credentials`(`id`)
  ON DELETE CASCADE
  ON UPDATE RESTRICT": syntax error at position 59 near 'foreign'
```